source/Applications/Advanced/Downsample/Downsample.cpp

```diff
@@ -10,12 +10,12 @@ Import a ZDF point cloud and downsample it.
 
 #include <iostream>
 
-Zivid::PointCloud downsample(const Zivid::PointCloud &, int);
-Eigen::MatrixXi downsampleAndRound(const Eigen::MatrixXi &, int);
-Eigen::MatrixXf gridSum(const Eigen::MatrixXf &, int);
-Eigen::MatrixXf lineSum(Eigen::MatrixXf, int);
-float nanToZero(float);
-void visualizePointCloud(const Zivid::PointCloud &, Zivid::Application &);
+Zivid::PointCloud downsample(const Zivid::PointCloud &pointCloud, int downsamplingFactor);
+Eigen::MatrixXi downsampleAndRound(const Eigen::MatrixXi &matrixi, int downsamplingFactor);
+Eigen::MatrixXf gridSum(const Eigen::MatrixXf &matrixf, int downsamplingFactor);
+Eigen::MatrixXf lineSum(Eigen::MatrixXf matrix, int downsamplingFactor);
```

Comment:
Add some consistency between the variable names - matrix on all, or matrixXF and matrixXI, or similar.

Reply:
I'll update to matrixf for the lineSum argument, ok?